src/Enemy.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Enemy.hpp"
+#include "Game.hpp"
+#include "Vector2.hpp"
+
+raylib::Vector2 Enemy::generateSpawnPosition() {
+	int spawnLimit { 100 };
+	int spawnX;
+	do {
+		spawnX = GetRandomValue(-SCREEN_WIDTH / 2, SCREEN_WIDTH / 2);
```

Comment:
We can change this so the enemies spawn in a circle around the player instead of randomly on the screen. I can implement it in the branch before we merge